packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -277,6 +279,20 @@ export class Calendar extends React.Component<ICalendarProps, ICalendarState> im
     }
   }
 
+  private _onBlur = (event: React.FocusEvent<HTMLElement>) => {
+    if (!event.currentTarget.contains(event.relatedTarget as Node)) {
```

Comment:
element.contains is not supported in IE11. you can use `elementContains` we have in utilities package. and make sure after you update the code you test this in IE11 too :)